fedmsg/config.py

```diff
@@ -294,8 +294,8 @@ class FedmsgConfig(dict):
             'validator': _validate_none_or_type(six.text_type),
         },
         'crl_cache_expiry': {
-            'default': None,
-            'validator': _validate_none_or_type(six.text_type),
+            'default': 10,
```

Comment:
The docs say this is in seconds (although the docs _are_  a little unreliable) so 10 is a really small default for a CRL. Maybe an hour is more reasonable?

Reply:
Good point. Updating...

Reply:
I think you missed updating it in a few places? Additionally, an hour would be 60*60 (3600) since this is seconds, not minutes.

Reply:
You don't want to to expire in 1 minute? 😂 
Let me try this again.

Reply:
Totally missed several spots.
Alright I think it finally got this.
Also, sorry for the constant rebasing. Just trying to keep clean commits.